.github/workflows/dynamoBinDiff.yml

```diff
@@ -0,0 +1,86 @@
+# Build Dynamo using latest VS and DotNET and perform a Bin Diff
+name: Dynamo-BinDiff
+on: [push,pull_request]
+jobs:
+ build-current:
+  runs-on: windows-2022
+  steps:
+    - name: Checkout Dynamo Repo current branch
```

Comment:
I have a complication for you - we also need to do this same comparison for the DynamoAll.Net6.Sln targeting windows.

Would you prefer to just duplicate this action for net6 or have this job do 4 builds?

Reply:
Added 2 more builds for Net6-Windows (parallel builds for quick results)

![Screenshot 2023-05-22 at 8 10 43 PM](https://github.com/DynamoDS/Dynamo/assets/32665108/160c6ba7-68df-44be-908a-d0458ad38d60)